rllib/examples/custom_eval.py

```diff
@@ -76,6 +76,7 @@
 from ray.rllib.utils.test_utils import check_learning_achieved
 
 parser = argparse.ArgumentParser()
+parser.add_argument("--evaluation-parallel-to-training", action="store_true")
```

Comment:
Add parallel option to this script and add setting this option to tests in BUILD.